reference/composedcubemaptexture.md

```diff
@@ -0,0 +1,125 @@
+## ComposedCubeMapTexture
+
+```
+ComposedCubeMapTexture {
+  vrmlField SFNode right   NULL
+  vrmlField SFNode left    NULL
+  vrmlField SFNode top     NULL
+  vrmlField SFNode bottom  NULL
+  vrmlField SFNode front   NULL
+  vrmlField SFNode back    NULL
+}
+```
+
+### Description
+
+The [ComposedCubeMapTexture](#composedcubemaptexture) node defines a cubic texture used for environment mapping. It is composed of exactly six [ImageTexture](imagetexture.md) nodes, all of which have to be defined for the resulting texture to be generated.
```

Comment:
I would add a sentence, like "If an ImageTexture node is missing, ..." to explain what happen in this case. By default all the textures are set to `NULL`, so it shouldn't crash in that case. So, what if a texture or all textures are missing. Is it using a while or black texture instead? Is the result undefined?